### PR TITLE
(refactor): remove kadira:dochead meteor package and add needed functions to a core plugin

### PR DESCRIPTION
**Pull request by willopez**
This PR removes the kadira:dochead meteor package, and adds a core plugins to provide the necessary functionality. A new namespace named `Reaction.DOM` is added to provide functions to manipulate the document.

In particular, `Reaction.DOM.setMetaTag` is used to add meta tags.

**Testing Procedure**
- Login as a shop administrator
- Click on the "Shop" link
- Under general add a description and keywords
- Reload page and use the inspect element to verify that meta tags were added

### CODE REVIEW

File: imports/plugins/core/dom/client/dom.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+// import { Tracker } from "meteor/tracker";
```
Comment (aldeed) [inline imports/plugins/core/dom/client/dom.js]: Remove this comment

Comment (willopez) [reply imports/plugins/core/dom/client/dom.js]: comment removed.

Comment (willopez) [reply imports/plugins/core/dom/client/dom.js]: and logic in metadata.js has been refactor to address your comment above.

**Comment by aldeed** [comment]
@willopez Looks great now! It looks like one test failed on CI, but I think that's a test that fails randomly sometimes. I approved.

**Comment by aaronjudd** [comment]
I hit the "rebuild" on the test -> you should always be able to do that.. .(we do block merges on failing tests..).. fingers crossed.

**Comment by willopez** [comment]
Yeah, something is up with that test. @aaronjudd I tried to click rebuild on circle CI, but I don't have the proper permissions.

**Comment by machikoyasuda** [comment]
@willopez - As a part of this PR and change, can you update https://docs.reactioncommerce.com/reaction-docs/master/seo-metadata for 1.8? I made an issue in Docs here: https://github.com/reactioncommerce/reaction-docs/issues/489

**Pull request merged**